src/views/Layout.jsx

```diff
@@ -12,11 +13,14 @@ import { auth } from '../api/config.js';
  */
 
 export function Layout() {
+	const { user } = useAuth();
+	console.log({ user });
```

Comment:
Do we want to leave this in for now?

Reply:
Ooops, It should be removed. I will update it soon